docs/Makefile

```diff
@@ -52,8 +52,13 @@ build/swagger.stamp: ../proto/build/swagger/determined/api/v1/api.swagger.json s
 	mkdir -p build
 	touch $@
 
+.PHONY: gen-versions
+gen-versions: ANCESTOR=$(shell git rev-parse $(shell git merge-base 0.21.0 main)~1)
```

Comment:
Oh, is that the epoch in terms of these versions?  If so, I guess that makes sense.  But maybe add a comment so it makes sense to the next person. :)

Reply:
Yup, I use the parent of that commit as the cutoff point for old versions, since the OG `versions.json` doesn't go back beyond that. I'll add a comment to clarify, though.